settings.py

```diff
@@ -4,11 +4,22 @@
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-store: %(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-MONGODB_URL = os.getenv("MONGODB_URL")
+DB_HOST = os.getenv('POSTGRES_HOST', '0.0.0.0')
+DB_PORT = os.getenv('POSTGRES_PORT', '5432')
+DB_NAME = os.getenv('POSTGRES_NAME', 'postgres')
+DB_USER = os.getenv('POSTGRES_USER', 'postgres')
+DB_PASSWORD = os.getenv('POSTGRES_PASSWORD', 'secret')
+DB_URI = 'postgresql://{}:{}@{}:{}/{}'.format(DB_USER,
+                                              DB_PASSWORD,
+                                              DB_HOST,
+                                              DB_PORT,
+                                              DB_NAME)
+
```

Comment:
Since @gavinleeedwards is removing defaults in his story, should we have default values here? or are you secretly trying to make more work for him ;)